### ISSUE TITLE
Intermediate link box too big

### ISSUE THREAD

**Issue by ego-lay-atman-bay** [ImranR98/Obtainium#1211]
Simple put, the imtermediate link box is too big, probably over 5x the screen height.

![Screenshot_20231229-094713](https://github.com/ImranR98/Obtainium/assets/70973578/9966c151-81a2-4d8d-bc59-bc6c6cc03c24)

https://github.com/ImranR98/Obtainium/assets/70973578/8f1d2fde-efc8-46f5-86d1-4ca246e9a4ba

 - Device: Pixel 6a
 - OS: Android 14
 - Obtainium Version: 0.15.0-beta

**Comment by ImranR98** [comment]
Could you share an export

**Comment by ego-lay-atman-bay** [comment]
Of what?

**Comment by ImranR98** [comment]
Of your apps list. From the import/export page.

**Comment by ego-lay-atman-bay** [comment]
Oh
```json
{"apps":[{"id":"dev.imranr.obtainium","url":"https://github.com/ImranR98/Obtainium","author":"ImranR98","name":"Obtainium","installedVersion":"v0.15.0-beta","latestVersion":"v0.15.0-beta","apkUrls":"[[\"app-arm64-v8a-release.apk\",\"https://api.github.com/repos/ImranR98/Obtainium/releases/assets/142845332\"]]","preferredApkIndex":0,"additionalSettings":"{\"includePrereleases\":true,\"fallbackToOlderReleases\":true,\"filterReleaseTitlesByRegEx\":\"\",\"filterReleaseNotesByRegEx\":\"\",\"verifyLatestTag\":false,\"dontSortReleasesList\":false,\"trackOnly\":false,\"versionDetection\":\"standardVersionDetection\",\"apkFilterRegEx\":\"\",\"autoApkFilterByArch\":true,\"appName\":\"\",\"exemptFromBackgroundUpdates\":false,\"skipUpdateNotifications\":false,\"about\":\"\"}","lastUpdateCheck":1703873223931097,"pinned":false,"categories":[],"releaseDate":1703811724000000,"changeLog":"## Notice\r\nThis release contains significant changes to background tasks and the HTML source. Please look out for any new issues or bugs.\r\n\r\n## What's Changed\r\n* Update Spanish 3rd pass by @CertainBot in https://github.com/ImranR98/Obtainium/pull/1197\r\n* Update pt.json by @LucasTavaresA in https://github.com/ImranR98/Obtainium/pull/1196\r\n* Update it.json by @unbranched in https://github.com/ImranR98/Obtainium/pull/1190\r\n* Update de.json by @iDazai in https://github.com/ImranR98/Obtainium/pull/1189\r\n* By @ImranR98 in https://github.com/ImranR98/Obtainium/pull/1204:\r\n  * Switch to a New Background Task Plugin (#608)\r\n  * Add Link Text Filter for HTML Links (#1182)\r\n  * Add Support for Multiple Intermediate Links to HTML Source \r\n\r\n**Full Changelog**: https://github.com/ImranR98/Obtainium/compare/v0.14.41-beta...v0.15.0-beta","overrideSource":null,"allowIdChange":false},{"id":"org.videolan.vlc","url":"https://videolan.org","author":"VideoLAN","name":"VLC","installedVersion":"3.5.4","latestVersion":"3.5.4","apkUrls":"[[\"VLC-Android-3.5.4-arm64-v8a.apk\",\"https://get.videolan.org/vlc-android/3.5.4/VLC-Android-3.5.4-arm64-v8a.apk\"]]","preferredApkIndex":0,"additionalSettings":"{\"trackOnly\":false,\"versionDetection\":\"standardVersionDetection\",\"apkFilterRegEx\":\"\",\"autoApkFilterByArch\":true,\"appName\":\"VLC\",\"exemptFromBackgroundUpdates\":false,\"skipUpdateNotifications\":false,\"about\":\"\"}","lastUpdateCheck":1703873222312139,"pinned":false,"categories":[],"releaseDate":null,"changeLog":null,"overrideSource":null,"allowIdChange":false},{"id":"com.best.deskclock","url":"https://github.com/qw123wh/new-clock","author":"qw123wh","name":"Clock","installedVersion":"test","latestVersion":"test","apkUrls":"[[\"app-Retail.apk\",\"https://api.github.com/repos/qw123wh/new-clock/releases/assets/73241599\"]]","preferredApkIndex":0,"additionalSettings":"{\"includePrereleases\":false,\"fallbackToOlderReleases\":true,\"filterReleaseTitlesByRegEx\":\"\",\"filterReleaseNotesByRegEx\":\"\",\"verifyLatestTag\":false,\"dontSortReleasesList\":false,\"trackOnly\":false,\"versionDetection\":\"noVersionDetection\",\"apkFilterRegEx\":\"\",\"autoApkFilterByArch\":true,\"appName\":\"\",\"exemptFromBackgroundUpdates\":false,\"skipUpdateNotifications\":false,\"about\":\"\"}","lastUpdateCheck":1703873222323612,"pinned":false,"categories":[],"releaseDate":1659270968000000,"changeLog":"Fix #25 ","overrideSource":null,"allowIdChange":true},{"id":"org.dolphinemu.dolphinemu","url":"https://dolphin-emu.org/download/?ref=btn","author":"dolphin-emu.org","name":"Dolphin Emulator","installedVersion":"5.0-20347","latestVersion":"5.0-20347","apkUrls":"[[\"https://dl.dolphin-emu.org/builds/60/44/dolphin-master-5.0-20347.apk\",\"https://dl.dolphin-emu.org/builds/60/44/dolphin-master-5.0-20347.apk\"]]","preferredApkIndex":0,"additionalSettings":"{\"intermediateLink\":[{\"customLinkFilterRegex\":\"\",\"filterByLinkText\":null}],\"customLinkFilterRegex\":\"https:\\\\/\\\\/dl\\\\.dolphin-emu\\\\.org\\\\/builds\\\\/[a-zA-Z0-9]+\\\\/[a-zA-Z0-9]+\\\\/dolphin-master-[0-9]\\\\.[0-9]-[0-9]+.apk$\",\"filterByLinkText\":false,\"skipSort\":true,\"reverseSort\":true,\"sortByLastLinkSegment\":true,\"versionExtractionRegEx\":\"[0-9]\\\\.[0-9]-[0-9]+\",\"matchGroupToUse\":\"\",\"versionExtractWholePage\":false,\"supportFixedAPKURL\":false,\"trackOnly\":false,\"versionDetection\":\"noVersionDetection\",\"apkFilterRegEx\":\"\",\"autoApkFilterByArch\":true,\"appName\":\"\",\"exemptFromBackgroundUpdates\":false,\"skipUpdateNotifications\":false,\"about\":\"\",\"sortByFileNamesNotLinks\":true,\"intermediateLinkRegex\":\"\"}","lastUpdateCheck":1703827089611913,"pinned":false,"categories":[],"releaseDate":null,"changeLog":null,"overrideSource":"HTML","allowIdChange":false}]}
```

It only appears to happen for dolphin

**Comment by ego-lay-atman-bay** [comment]
Alright it's fixed in 0.15.1-beta
